src/main/java/org/ibp/api/java/germplasm/GermplasmService.java

```diff
@@ -38,4 +39,9 @@
 	long countGermplasmByStudy(final Integer studyDbId);
 
 	List<GermplasmDTO> getGermplasmByStudy(int studyDbId, int pageSize, int pageNumber);
+
+	List<AttributeDTO> getAttributesByGid(
```

Comment:
@vmaletta Remove final from the interface function params

Reply:
ok